stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptions.kt

```diff
@@ -129,90 +129,159 @@ public fun defaultMessageOptionsState(
     val isMessageFailed = selectedMessage.syncStatus == SyncStatus.FAILED_PERMANENTLY
 
     return listOfNotNull(
```

Comment:
I think this change is also less optimized - as we'll have X more composable slots in the UI tree that recompose every time the state changes, rather than one slot that recomposes everything.

But I'll have to check and test

Reply:
It depends on how it adds the item to the UI tree - but it's the same, yes, tested! :]

It's just a matter of 1 function being a composable and X of them, but in the end it seems to be optimized anyway